full-node/sov-stf-runner/src/runner.rs

```diff
@@ -11,19 +11,18 @@ use tokio::sync::oneshot;
 use tracing::{debug, info};
 
 use crate::verifier::StateTransitionVerifier;
-use crate::{RunnerConfig, StateTransitionData};
-
+use crate::{ProverService, RunnerConfig, StateTransitionData};
 type StateRoot<ST, Vm, Da> = <ST as StateTransitionFunction<Vm, Da>>::StateRoot;
 type InitialState<ST, Vm, Da> = <ST as StateTransitionFunction<Vm, Da>>::GenesisParams;
 
 /// Combines `DaService` with `StateTransitionFunction` and "runs" the rollup.
-pub struct StateTransitionRunner<Stf, Sm, Da, Vm, V>
+pub struct StateTransitionRunner<Stf, Sm, Da, Vm, PS>
 where
     Da: DaService,
     Vm: ZkvmHost,
     Sm: StorageManager,
     Stf: StateTransitionFunction<Vm, Da::Spec, Condition = <Da::Spec as DaSpec>::ValidityCondition>,
-    V: StateTransitionFunction<Vm::Guest, Da::Spec>,
+    PS: ProverService,
```

Comment:
Nit: Our naming convention for abbreviations is title case, could you please make `Ps`